draft-vanbrouwershaven-acme-auto-discovery.md

```diff
@@ -114,16 +114,16 @@ example.com CAA 0 issue "ca.example"
 
 By default, when multiple CAA records are present, the CAs are randomized to distribute the load. However, some users may have preferences regarding the order in which CAs are attempted for certificate issuance. To explicitly specify the order, the "priority" parameter can be used.
 
-In the next example, the domain "example.com" has two CAA records. The CAA record with "ca2.example" has a higher priority value of 2, indicating it should be attempted first. The CAA record with "ca1.example" has a lower priority value of 1, indicating it should be attempted second.
+In the next example, the domain "example.com" has two CAA records. The CAA record with "ca1.example" has a higher priority value of 1, indicating it should be attempted first. The CAA record with "ca2.example" has a lower priority value of 2, indicating it should be attempted second.
 
 ~~~ dns-rr
-example.com CAA 0 issue "ca1.example; priority=2"
-example.com CAA 0 issue "ca2.example; priority=1"
+example.com CAA 0 issue "ca1.example; priority=1"
+example.com CAA 0 issue "ca2.example; priority=2"
 ~~~
```

Comment:
In the example ca1.example has priority 2, not 1. This was done on purpose to not confuse the case name with the priority value.

Reply:
Flipped it back!